### DISCUSSION TITLE
how to insert a column right/left ?

### DISCUSSION THREAD

**Discussion by gorman93** [exceljs/exceljs#2481] [Q&A]

**Comment by DonnaWuDongxia** [comment]
also insterested

**Comment by DonnaWuDongxia** [comment]
I made a walkaround by following:

function addBlankColumns(count, table) {
  let blankCoumns = [];
  for (let i = 0; i < count; i++) {
    blankCoumns.push({
      header: null,
      key:  \`__blank${i}\`
    });
  }
  table.columns = blankCoumns.concat(table.columns);
}
